projects/plugins/jetpack/extensions/blocks/map/deprecated/v1/index.js

```diff
@@ -69,6 +74,7 @@ export default {
 			.trim();
 		return {
 			...omit( oldAttributes, 'mapStyle' ),
```

Comment:
Now might be a good time to replace the use of lodash's `omit` here with vanilla JS.

Reply:
Is there any detail anywhere about whether we are trying to slowly remove lodash completely, or are we just trying to minimise its usage to just utility methods that are not available in vanilla js? 

I can see the benefit of just using plain map,filter,reduce, etc. where there is a clear 1-1 mapping with vanilla js, but with `omit` you either end up having to write your own utility function or use something like:

```js
 const newAttributes = (({ prop1, prop2, ...rest }) => rest)(attributes);
```
which isn't as quick to grock what is happening at first glance. I am in favour of retaining lodash utilities for cases like this, but I may be missing a vanilla js way of doing it that is just as succinct and readable.

Reply:
I don't have a strong opinion either way though

Reply:
There was a write-up about slowly removing some lodash methods (those where a native approach exists) a few weeks back in a Calypso context, but Jetpack was involved too p4TIVU-9Bf-p2

Reply:
The only link I have at hand quickly is this discussion: p4TIVU-9Bf-p2

I also think you can make the above easier to grok without having to write the utility function.

```js
const { excludedProp1, excludedProp2, ...newAttributes } = attributes;
```

Reply:
I knew there would be an easier way I was overlooking

Reply:
except you probably then need to add a lint ignore rule to stop it complaining about the excluded constants you defined but then didn't use 😄

Reply:
@aaronrobertshaw that seems to work, have pushed that change. It is weird that eslint doesn't complain about mapStyle being defined in the destructure but never used?

Reply:
I haven't looked at the specific config within Jetpack but the `no-unused-vars` eslint rule has an `ignoreRestSiblings` option, which if set to true, would ignore the `mapStyle` in this example. Pretty much tailor made for this instance.